nekoyume/Assets/_Scripts/UI/Module/Item/ItemView.cs

```diff
@@ -185,6 +185,43 @@ public void SetData(TViewModel model, bool isConsumable)
             UpdateView();
         }
 
+        public void SetDataExceptOptionTag(TViewModel model)
```

Comment:
Does it disable existing option tag enabled before?

Reply:
This method has few lines like below differ with `void SetDate(TViewMedel)`.

```
            var tagData = optionTagData.GetOptionTagData(row.Grade);
            Model.HasOptions.Subscribe(hasOptions => SetOptionTag(hasOptions, tagData))
                .AddTo(_disposablesAtSetData);
```

Could this refactor like below?

```
1. public override void SetData(TViewModel) -> public void SetData(TViewModel, bool)

2. public void SetDataExceptOptionTag(TViewModel) -> delete and use above method.
```

Reply:
There are cases where hasOptions is true , but the view is disabled. like the screenshot below
![image](https://user-images.githubusercontent.com/40553495/134839503-d10e41f9-25d3-40ae-9ebd-ad03b7cf66b0.png)